### DISCUSSION TITLE
Phoenix LiveView and Views - Post

### DISCUSSION THREAD

**Discussion by dbernheisel** [dbernheisel/bernheisel.com#6] [General]
![phoenix-liveview-and-views](https://user-images.githubusercontent.com/643967/103720244-f0e38d00-4f98-11eb-9421-411b274325a4.png)

https://bernheisel.com/blog/phoenix-liveview-and-views

Have reactions to the post? Let's discuss!

**Comment by eldano** [comment]
Great post! Thanks for including all these examples that are really well explained and it's not easy to figure out from the docs.
It bugs me that by default one needs to restructure all the templates just to use an html helper. Thankfully there's that workaround that you presented. I still think the framework should provide a better way of doing this.

**Reply by eldano** [reply]
After saying this I changed my way and now use collocated templates so all my templates lie next to my components/liveviews. All the HTML functions that these templates use I'm defining them in the component or liveview and calling them in the template with `Module.function`. That way I don't need to define a render function in the component.
